test/support/env_mock_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Overwrite .env.sample default variables in tests to skip checks in models
+[
+  "JWPLAYER_API_KEY",
+  "JWPLAYER_API_V1_SECRET",
+  "JWPLAYER_API_V2_SECRET",
+  "MUX_API_KEY",
+  "MUX_API_ID",
+  "MUX_API_SECRET",
+].each do |env_variable|
+  ENV[env_variable] = "mock-#{env_variable}" if ENV[env_variable] == "find-me-in-vault"
+end
```

Comment:
Tohle mi prijde jako zbytecny ruby konstrukt. Radeji prosim:

1. vyhodme  `find-me-in-vault` z env sample a dejme `skip` do testu potrebujici tenhle `ENV` v pripade, ze ho nemam
2. nebo vyhodme ten `find-me-in-vault` gsub - realne to zadnou funkci neplni

Reply:
Když už, tak jsem pro  2) . K něčemu to tam bylo, ale už si nevzpomínám proč.